pkg/config/tlscfg/flags.go

```diff
@@ -21,13 +21,16 @@ import (
 )
 
 const (
-	tlsEnabled     = "tls"
-	tlsCA          = "tls.ca"
-	tlsCert        = "tls.cert"
-	tlsKey         = "tls.key"
-	tlsServerName  = "tls.server-name"
-	tlsClientCA    = "tls.client-ca"
-	tlsClientCAOld = "tls.client.ca"
+	tlsPrefix         = ".tls"
```

Comment:
I am not seeing changes in the modules (grpc, etc) that are already using this package and don't assume that it adds a dot prefix. Seems like they will be broken by this change.